packages/@aws-cdk/aws-lambda/lib/alias.ts

```diff
@@ -193,6 +201,39 @@ export class Alias extends QualifiedFunctionBase implements IAlias {
     });
   }
 
+  /**
+   * Enable autoscaling for the given lambda version. Requires that lambda version has provisioned concurrency.
```

Comment:
shouldn't this be "alias" and not "version" for these comments?

Reply:
This method docs should include what is being scaled (Provisioned Concurrency) and which metric is tracked (LambdaProvisionedConcurrencyUtilization). Add a link to the lambda performance metrics [docs](https://docs.aws.amazon.com/lambda/latest/dg/monitoring-metrics.html#monitoring-metrics-concurrency)

Reply:
Removed version in the docs and added link to lambda performance metrics.